projects/spirv-tools/generate_spirv_corpus.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2024 Google LLC
```

Comment:
I think you need to `COPY` this file in your dockerfile to use it.

Reply:
Thanks - done.